Makefile

```
@@ -14,7 +14,8 @@
 VERSION=11.0.0-dev
 
 DOCKER_IMAGE ?= quay.io/gravitational/teleport
-DOCKER_IMAGE_CI ?= quay.io/gravitational/teleport-ci
+DOCKER_IMAGE_ECR ?= public.ecr.aws/gravitational/teleport
+DOCKER_IMAGE_CI ?= 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport
```

```suggestion
DOCKER_IMAGE_STAGING ?= 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport
```

The CI nomenclature is confusing.  It should be CD if anything, but even then, we already have `staging` as a offical term for pre-promotion repositories.

---

https://github.com/gravitational/teleport/pull/14347/commits/7028c79eb1bb7788641df7b80a1030a88f8dba9c